packages/expo-dev-launcher/ios/EXDevLauncherController.h

```diff
@@ -1,7 +1,9 @@
 #import <React/RCTBridgeModule.h>
 #import <React/RCTBridgeDelegate.h>
 
+#if !TARGET_OS_OSX
 #import <UIKit/UIKit.h>
+#endif
```

Comment:
afaik we can just do this
```suggestion
#import <ExpoModulesCore/Platform.h>
```